### ISSUE TITLE
DOPs city of Rostock

### ISSUE THREAD

**Issue by kvlahromei** [hotosm/oam-uploader#61]
Hi,
I try for some time to contribute the aerial imagery [from our opendata portal](https://www.opendata-hro.de/dataset?tags=Luftbild) to OAM:

Unfortunately, also my second attempt also got stuck: https://upload.openaerialmap.org/#/status/58a15d37269a040011c88aba

Can anybody plz. point me on how to contribute all the material in an efficient way? It doesn't feel right, if I have to submit a dataset again and again to debug the processing and to spam your queue 😕

**Comment by smit1678** [comment]
@kvlahromei I looked at the error output and it looks like there are two issues at hand. First one is that you have submitted a folder instead individual files. The system ingests individual files one at a time and so you need to provide URLs directly to the files. Instead of `https://ftp.sv.rostock.de/public/folder/pzscWPxgEUWRDhYv5m2dMw/DOPs`, it should be `https://ftp.sv.rostock.de/public/folder/pzscWPxgEUWRDhYv5m2dMw/DOPs/luftbild_2000.ecw`. 

Second one is that I need to investigate our handling of `.ecw` files. I ran a quick test on that image and it had an error. We should be able to accept any type of GDAL source, so either the file has an error or our system doesn't handle that specific flavor of `ecw`. 

We'll investigate and post back here. 

A quick fix here would be to provide geotiff or jp2 forms of the files. Do you have the files in these other formats?

**Comment by kvlahromei** [comment]
Thanks for your reply. I guess it makes sense to (pre)process also metadata that are usually tied to an external file (worldfile). Should I open a seperate issue for your roadmap?

Sorry, we just host the .ECW files and offer the imagery via WMS (no TMS / WMS-T)
https://geo.sv.rostock.de/geodienste/luftbild_2000/wms?service=WMS&version=1.3.0&request=GetCapabilities

**Comment by smit1678** [comment]
> I guess it makes sense to (pre)process also metadata that are usually tied to an external file (worldfile).

Yes agreed and that's partly why we've initially pushed for formats that have metadata directly in the file like geotiff. Formats that require multiple files are more technically challenging but this is where you're comment about zipped files comes into play. 

> Should I open a seperate issue for your roadmap?

@kvlahromei Yes, that would be great. We're planning some additional development this spring and it would be great to make sure we use your workflow and files as a primary use case.

Is the file here https://ftp.sv.rostock.de/public/folder/pzscWPxgEUWRDhYv5m2dMw/DOPs the only file you are looking to integrate? Or are there additional files?

**Comment by kvlahromei** [comment]
I would like to contribute all DOPs which are listed at the link in the first post. All Luftbilder* files can be downloaded easily via https://geo.sv.rostock.de/download/opendata/

**Comment by kvlahromei** [comment]
Any progress here? Should I recontribute the DOPs as single .ecw files or does the file handling error persist?

**Comment by smit1678** [comment]
@kvlahromei We haven't been able to test ECW support within the system yet. A temp workaround would be to convert to geotiff or JP2 and then upload the individual files.

**Comment by kvlahromei** [comment]
I retried the submit of "luftbilder 2000", but after the upload reports an general error.

Do you see any other way, how we can share our DOPs to your project?
